src/main/target/BLUEJAYF4/target.c

```diff
@@ -31,4 +34,15 @@ const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
     { TIM3, IO_TAG(PB1), TIM_Channel_4, TIM_USE_MOTOR | TIM_USE_LED, 1, GPIO_AF_TIM3, DMA1_Stream2, DMA_Channel_5, DMA1_ST2_HANDLER }, // S5_OUT
     { TIM3, IO_TAG(PB0), TIM_Channel_3, TIM_USE_MOTOR,               1, GPIO_AF_TIM3, DMA1_Stream7, DMA_Channel_5, DMA1_ST7_HANDLER }, // S6_OUT
 };
+*/
+
+const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = { 
+    DEF_TIM(TIM8, CH2, PC7, TIM_USE_PPM,                 0, 0 ), // PPM IN
```

Comment:
This could be possible, and I did think of it - but it meant duplicating a heap ... i.e. CH2N versus CH2 for deciding if the TIMER_OUTPUT_N_CHANNEL should or shouldn't be included.

You effectively have to create a definition for 
```
xx_CHAN_TIM8_CH2     TIM_Channel_2
xx_CHAN_TIM8_CH2N    TIM_Channel_2
xx_NCHN_TIM8_CH2     NONE
xx_NCHN_TIM8_CH2N    TIMER_OUTPUT_N_CHANNEL   
```

but for every possibly combo. Where as separating the two means you only define the 8 possibles for CH1..4 and CH1N..4N

Though my preprocessing macro skills are not the best so if there are some parsing / substr like capability - then I guess that could occur?